index.html

```diff
@@ -17,102 +17,120 @@
-      <button class="see-project-btn full-width-btn">See Project</button>
-    </div>
-    <div class="projects not-featured">
-      <h2 class="project-title not-featured-title">Profesional Art Printing Data</h2>
-      <p class="project-description not-featured-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="used-tools tools-bottom">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="see-project-btn full-width-btn">See Project</button>
+  <div class="top-header">
+    <h1 class="recent-title">My Recent Works</h1>
+    <div class="separate-line">
+      <hr/>
     </div>
-    <div class="projects not-featured">
-      <h2 class="project-title not-featured-title">Profesional Art Printing Data</h2>
-      <p class="project-description not-featured-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="used-tools tools-bottom">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="see-project-btn full-width-btn">See Project</button>
-    </div>
-    <div class="projects not-featured">
-      <h2 class="project-title not-featured-title">Profesional Art Printing Data</h2>
-      <p class="project-description not-featured-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="used-tools tools-bottom">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="see-project-btn full-width-btn">See Project</button>
-    </div>
-    <div class="projects not-featured">
-      <h2 class="project-title not-featured-title">Profesional Art Printing Data</h2>
-      <p class="project-description not-featured-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-      <ul class="used-tools tools-bottom">
-        <li>html</li>
-        <li>bootstrap</li>
-        <li>Ruby</li>
-      </ul>
-      <button class="see-project-btn full-width-btn">See Project</button>
+  </div>
+  <section class="work-section">
+    <div class="projects featured pro-top">
+      <div class="top-img">
+        <img src="./images/ImgPlaceholder.png" alt="featured project" class="first-project">
+      </div>
+      <div class="pro-detail">
+        <h2 class="project-title">Multi-Post Stories</h2>
+        <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="used-tools">
+          <li>css</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
```

Comment:
- [ ] [ OPTIONAL ] Kindly apply the right Font to these list items as specified in the Figma design(Inter). Remember to make this change on all the cards in this section.